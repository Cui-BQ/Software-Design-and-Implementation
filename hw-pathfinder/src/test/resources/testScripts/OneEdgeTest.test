CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2

AddEdge graph1 n1 n2 2.0
AddEdge graph1 n1 n2 0.2

FindPath graph1 n1 n2
