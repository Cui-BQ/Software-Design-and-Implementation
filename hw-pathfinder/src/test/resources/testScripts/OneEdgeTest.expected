created graph graph1

added node n1 to graph1
added node n2 to graph1

added edge 2.000 from n1 to n2 in graph1
added edge 0.200 from n1 to n2 in graph1

path from n1 to n2:
n1 to n2 with weight 0.200
total cost: 0.200